Uninitialized variable in ja4_fingerprint plugin

I noticed this when testing a ubuntu build with docker:

```
/work/plugins/experimental/ja4_fingerprint/plugin.cc: In function 'constexpr TSPluginRegistrationInfo get_registration_info()':
/work/plugins/experimental/ja4_fingerprint/plugin.cc:98:28: error: uninitialized variable 'info' in 'constexpr' function
   98 |   TSPluginRegistrationInfo info;
      |                            ^~~~
In file included from /work/plugins/experimental/ja4_fingerprint/plugin.cc:27:
/work/build-ci/include/ts/apidefs.h:160:8: note: 'struct TSPluginRegistrationInfo' has no user-provided default constructor
  160 | struct TSPluginRegistrationInfo {
      |        ^~~~~~~~~~~~~~~~~~~~~~~~
/work/build-ci/include/ts/apidefs.h:161:15: note: and the implicitly-defined constructor does not initialize 'const char* TSPluginRegistrationInfo::plugin_name'
  161 |   const char *plugin_name;
      |               ^~~~~~~~~~~
/work/plugins/experimental/ja4_fingerprint/plugin.cc: In function 'bool register_plugin()':
/work/plugins/experimental/ja4_fingerprint/plugin.cc:131:45: error: 'constexpr TSPluginRegistrationInfo get_registration_info()' called in a constant expression
  131 |   constexpr auto info{get_registration_info()};
      |                                             ^
```

--- a/plugins/experimental/ja4_fingerprint/plugin.cc
+++ b/plugins/experimental/ja4_fingerprint/plugin.cc
@@ -95,7 +95,7 @@ get_log_handle()
 static constexpr TSPluginRegistrationInfo
 get_registration_info()
 {
-  TSPluginRegistrationInfo info;
+  TSPluginRegistrationInfo info{};
   info.plugin_name   = PLUGIN_NAME;
   info.vendor_name   = PLUGIN_VENDOR;
   info.support_email = PLUGIN_SUPPORT_EMAIL;
@@ -179,7 +179,7 @@ handle_client_hello(TSCont /* cont ATS_UNUSED */, TSEvent event, void *edata)
 std::string
 get_fingerprint(SSL *ssl)
 {
-  JA4::TLSClientHelloSummary summary;
+  JA4::TLSClientHelloSummary summary{};
   summary.protocol    = JA4::Protocol::TLS;
   summary.TLS_version = get_version(ssl);
   summary.ALPN        = get_first_ALPN(ssl);

